### DISCUSSION TITLE
Supporting display a higher Sparkline?

### DISCUSSION THREAD

**Discussion by labo-China** [Textualize/textual#4146] [Ideas]
I am currently working on a music player app and I would like to visualize the music by displaying the audio spectrum in a Sparkline widget like the picture below:
![image](https://github.com/Textualize/textual/assets/62234752/19438dc0-96a4-4e07-85f3-6a19c74a96ed)
Unluckily, I looked through the source code and found that it only support 1-line height.
I guess it`s possible to support displaying a higher Sparkline by rendering in multiple lines. Would you consider adding it in the future?

B.T.W: May you see the issue #4141? It`s causing my whole app flickering! Thanks for your efforts.

**Comment by davep** [comment]
While it's not my place to say yes/no to such a request, I would point out that it's pretty much a defining characteristic of a sparkline that it fits within a single line. I would imagine though that it wouldn't be too difficult to make the sort of widget you're envisaging by starting out with `Sparkline`.
